### PR TITLE
Argument check for lambdify() . Issue #9122

### PR DESCRIPTION
**Pull request by meghana-madhyastha**
Checking arguments for lambdify(). Issue #9122

### CODE REVIEW

**Comment by aktech** [comment]
Can you please change the title of the PR? the number B3 doesn't tell the reviewer what they are going to see. 

:eyeglasses: See the section [Writing pull request title and description](https://github.com/sympy/sympy/wiki/Development-workflow#writing-pull-request-title-and-description) at the Development Workflow.

**Comment by debugger22** [comment]
Indent with spaces instead of tabs.

**Comment by debugger22** [comment]
See [this](https://help.github.com/articles/closing-issues-via-commit-messages/) too.

File: sympy/utilities/lambdify.py [hunk 1]
```diff
@@ -296,8 +296,13 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     in other namespaces, unless the ``use_imps`` input parameter is False.
     """
     from sympy.core.symbol import Symbol
+    from sympy.core import sympify
     from sympy.utilities.iterables import flatten
-
+    # If the user hasn't specified any modules, use what is available.
+    if not sympify(expr) == expr:
+	raise TypeError("expr must be a valid sympy expression")
+    if not ((isinstance(args, list)) or (isinstance(args, Symbol)) or (isinstance(args, tuple))):
```
Comment (debugger22) [inline sympy/utilities/lambdify.py]: How about this?
```
if not isinstance(args, (Symbol, list, tuple)):
```

**Comment by debugger22** [comment]
You don't need to close a PR everytime you want to update it. Just push to the same branch and the change will reflect here.

**Comment by meghana1995** [comment]
oh ok, didn't realize that. Sorry for the inconvenience.
I made the changes you asked (Changed the tabs to spaces and modified the
isinstance line).
Thank you for your suggestions and help !

**Pull request closed without merging**